.github/labeler.yml

```diff
@@ -0,0 +1,45 @@
+"area:help centre":
+  - changed-files:
+      - any-glob-to-any-file:
+          - wiki/Help_centre/**
+
+"area:legal":
+  - changed-files:
+      - any-glob-to-any-file:
+          - wiki/Legal/**
+
+"area:meta":
+  - changed-files:
+      - any-glob-to-any-file:
+          - "*"
```

Comment:
this will mark everything as `area:meta`, no?

Reply:
only files in root

Reply:
only files in root